python/ray/_private/worker.py

```diff
@@ -1303,6 +1303,12 @@ def init(
 
     if local_mode:
         driver_mode = LOCAL_MODE
+        warnings.warn(
+            "DeprecationWarning: local mode is an experimental feature that is no "
+            "longer maintained and will be removed in the future.",
```

Comment:
Shall we point user to the Ray debugger, since local_mode was used for debugging?

Reply:
Adding mentioning of the debugger, ptal?